scala/scala.bzl

```diff
@@ -162,9 +162,31 @@ def _compile(ctx, cjars, dep_srcjars, buildijar):
     all_srcjars = set(srcjars + list(dep_srcjars))
     # look for any plugins:
     plugins = _collect_plugin_paths(ctx.attr.plugins)
+
+    if (hasattr(ctx.attr, 'enable_dependency_analyzer')
+        and ctx.attr.enable_dependency_analyzer
```

Comment:
The feature to turn off dependency analyzer has not been added yet. This if statement is used because `compile` method is used by `scala_binary` which does not support the plugin yet, and also the bootstrapping version of `scala_library`. What test do you suggest can be added here?